deploy/olm-catalog/ibm-platform-api-operator/3.6.0/ibm-platform-api-operator.v3.6.0.clusterserviceversion.yaml

```diff
@@ -20,8 +20,9 @@ metadata:
                 "logrotateConf": "include /etc/logrotate.d"
               },
               "image": {
+                "imageRegistry": "quay.io/opencloudio",
                 "pullPolicy": "IfNotPresent",
-                "repository": "quay.io/opencloudio/icp-audit-service",
+                "repository": "icp-audit-service",
```

Comment:
how will the registry be combined with the image name and tag? Should that be done in the Makefile?

Reply:
i think we only need to deal with operand images, not operator image